NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/SetDefaultGroup.java

```diff
@@ -44,8 +44,7 @@ public boolean execute(CommandSender sender, String[] args) {
 			return true;
 		}
 		
-		GroupPermission gPerm = gm.getPermissionforGroup(g);
-		if (!gPerm.isAccessible(pType, PermissionType.BLOCKS)){
+		if (!gm.hasAccess(g.getName(), uuid, PermissionType.getPermission("BLOCKS"))){
```

Comment:
Yeah NameLayer wont know what it is and the perm check will just always return false if Citadel isn't enabled. I think this was put into place before, because the only use of default groups was reinforcements, so the permission for that could as well have been used.

I didnt remove this, because I wasn't sure whether I might disable a needed security mechanism with it, but after reworking the permissions for Citadel, I *think* we can remove this without any issues.